### DISCUSSION TITLE
User and password to acces cup

### DISCUSSION THREAD

**Discussion by Vctrsnts** [sergi0g/cup#51] [General]
Hi,
I have cup on VPS and is open for all the world and this dont like :(

Is possible to configure user and password to acces the web? 

Thanks

**Comment by sergi0g** [comment]
Hello! Currently, authentication isn't supported, but it's planned for a future version. As a workaround you can configure a reverse proxy with a route that requires authentication for Cup. Projects like [Authentik](https://github.com/goauthentik/authentik) can help with that.
